pcsx2-qt/Settings/GraphicsSettingsWidget.cpp

```diff
@@ -524,7 +524,7 @@ GraphicsSettingsWidget::GraphicsSettingsWidget(SettingsWindow* dialog, QWidget*
 			   "Fixes some processing effects such as the shadows in the Jak series and radiosity in GTA:SA."));
 
 		dialog->registerWidgetHelp(m_ui.disableDepthEmulation, tr("Disable Depth Emulation"), tr("Unchecked"),
-			tr("Disable the support of depth buffers in the texture cache."
+			tr("Disable the support of depth buffers in the texture cache. "
```

Comment:
Not sure why there is empty space added.

Reply:
This is how it actually looks like in the final product:
![imagen](https://github.com/PCSX2/pcsx2/assets/5641681/0e835912-970b-4569-8f7e-55c3b12d2c37)